src/stories/Header.stories.js

```diff
@@ -0,0 +1,17 @@
+import Header from '../components/Header';
+
+export default {
```

Comment:
Can you explain what's going on in this file? Maybe a link to some docs? Should we add some instructions to the `STYLE.md` style guide for creating `*.stories.js` files?

Reply:
Up for that, but might choose to do it in a follow up. There's a lot of documentation about Storybook available so I'm not sure what would go the style guide at this point except maybe something like "head to the project site and learn more about how to create stories".

Reply:
Added some links to the docs. Could do something more substantial eventually.